layers/+spacemacs/spacemacs-project/packages.el

```diff
@@ -54,6 +54,11 @@
             projectile-known-projects-file (concat spacemacs-cache-directory
                                                    "projectile-bookmarks.eld"))
       (spacemacs/set-leader-keys
+        ;; File path
+        "fyL" 'spacemacs/projectile-show-and-copy-file-relative-path-with-line-number
+        "fyc" 'spacemacs/show-and-copy-file-absolute-path-with-line-and-column-number
```

Comment:
Seems like `layers/+spacemacs/spacemacs-defaults/keybindings.el` should have this binding, and `layers/+spacemacs/spacemacs-project/packages.el` should have the `spacemacs/projectile-show-and-copy-file-relative-path` binding.